manticore/platforms/linux.py

```diff
@@ -1004,14 +1005,20 @@ def sys_lseek(self, fd, offset, whence):
         :return: 0 (Success), or EBADF (fd is not a valid file descriptor or is not open)
 
         '''
+        addressbitsize = {'i386':32, 'x86_64':64, 'armv71': 32}[self._uname_machine]
+        if addressbitsize == 32:
```

Comment:
The canonical way to access the width of the machine from the platform is via `self.current.address_bit_size`